app/main.py

```diff
@@ -1,2 +1,52 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self, capacity: int = 8) -> None:
+        self.capacity = capacity
+        self.size = 0
+        self.hash_table = [[] for _ in range(self.capacity)]
```

Comment:
Better fill out initial table with None values - no need to create 1 million empty lists.